docs/pages/configuration/dev/files/sync.mdx

```diff
@@ -240,18 +308,45 @@ dev:
 If multiple sync paths with `startContainer` exist, DevSpace will wait for all to finish before starting the container. You can combine `startContainer` sync configurations and non `startContainer` configurations to customize when the container is actually started.
 :::
 
+>>>>>>> upstream/v6-docs
 ### `onUpload.restartContainer`
 The `restartContainer` option expects a boolean which defines if DevSpace should restart the container every time either a single file or even a batch of files have been uploaded to the container using file sync.
 
 :::caution Restart Helper Required
+<<<<<<< HEAD
+Setting `restartContainer: true` requires to set `injectRestartHelper: true` for the image that is used to run the affected container. Otherwise, this option does not have any effect.
```

Comment:
```suggestion
Setting `restartContainer: true` requires you to also set `injectRestartHelper: true` for the image used to run the affected container. Otherwise, this option has no effect.
```

Reply:
@LukasGentele, I think I made a mistake in merging and accidentally kept this conflict in. Is this the correct section, or should it be 

```:::caution Restart Helper Required
Setting `restartContainer: true` requires you to also set `command: ["my", "container", "entrypoint"]`. Otherwise, DevSpace doesn't know which command to restart.
:::
```

Reply:
Yes. The quoted section you show here is the correct one. The `injectRestartHelper` option for images is being removed.